src/Microsoft.VisualStudio.AppDesigner/ApplicationDesigner/ApplicationDesignerView.vb

```diff
@@ -200,7 +200,6 @@ Namespace Microsoft.VisualStudio.Editors.ApplicationDesigner
                         '(Note: we don't expect this code path anymore, as I believe the project designer 
                         '  is always set up using the solution hierarchy because of the fact that it's
                         '  an editor on the project file.)
-                        Debug.Fail("GetNestedHierarchy failed")
```

Comment:
May as well just remove the whole `Else` branch.

Reply:
Good point, done.